internal/rpc/rpcserver/server.go

```diff
@@ -3361,6 +3355,59 @@ func marshalDecodedTxOutputs(mtx *wire.MsgTx, chainParams *chaincfg.Params) []*p
 	return outputs
 }
 
+// api.proto:
+//
+// enum ScriptClass {
+// 	NON_STANDARD = 0;
+// 	PUB_KEY = 1;
+// 	PUB_KEY_HASH = 2;
+// 	SCRIPT_HASH = 3;
+// 	MULTI_SIG = 4;
+// 	NULL_DATA = 5;
+// 	STAKE_SUBMISSION = 6;
+// 	STAKE_GEN = 7;
+// 	STAKE_REVOCATION = 8;
+// 	STAKE_SUB_CHANGE = 9;
+// 	PUB_KEY_ALT = 10;
+// 	PUB_KEY_HASH_ALT = 11;
+// 	TGEN = 12;
+// 	TADD = 13;
+
+func scProto(class stdscript.ScriptType) int32 {
```

Comment:
Previously it was assumed that the txscript script classes matched the protobuf enum.  To keep the gRPC results the same, we have to translate from the stdscript.ScriptType.